auth/src/main/java/com/vimeo/networking2/internal/AuthService.kt

```diff
@@ -327,9 +407,25 @@ internal interface AuthService {
     ): VimeoCall<Unit>
 
     companion object {
-        private const val AUTHORIZATION = "Authorization"
-        private const val SCOPE = "scope"
-        private const val GRANT_TYPE = "grant_type"
+        const val AUTHORIZATION = "Authorization"
+        const val SCOPE = "scope"
+        const val GRANT_TYPE = "grant_type"
+        const val REDIRECT_URI = "redirect_uri"
+        const val STATE = "state"
+        const val MARKETING_OPT_IN = "marketing_opt_in"
+        const val AUTHORIZATION_CODE = "authorization_code"
+        const val NAME = "name"
+        const val EMAIL = "email"
+        const val PASSWORD = "password"
+        const val TOKEN = "token"
+        const val TOKEN_SECRET = "token_secret"
+        const val DOMAIN = "domain"
+        const val USER_CODE = "user_code"
+        const val DEVICE_CODE = "device_code"
+        const val ACCESS_TOKEN = "access_token"
+        const val CODE = "code"
+        const val CLIENT_ID = "client_id"
+        const val ID_TOKEN = "id_token"
+        const val USERNAME = "username"
```

Comment:
In practice, we only ever pass an email to these functions, and historically we have called the parameters "email," but since the API calls it "username" and accepts some weird edge cases other than emails, I'll just rename all the parameters annotated by this to "username."